gokart/info.py

```diff
@@ -1,52 +1,23 @@
 import warnings
 from logging import getLogger
+from typing import List, NamedTuple, Optional, Set, Tuple
 
 import luigi
 
 from gokart.task import TaskOnKart
+from gokart.tree.task_info import make_tree_info_string
 
 logger = getLogger(__name__)
 
 
-def make_tree_info(task, indent='', last=True, details=False, abbr=True, visited_tasks=None):
+def make_tree_info(task, indent='', last=True, details=False, abbr=True, visited_tasks=None, ignore_task_names=None):
     """
     Return a string representation of the tasks, their statuses/parameters in a dependency tree format
+
+    This function has moved to `gokart.tree.task_info.make_tree_info_string`.
+    This code is remained for backward compatibility.
```

Comment:
[IMO] Since the number of arguments is getting larger, it is useful for the users of this functions that adding descriptions of arguments on docstring.

for example
https://github.com/pandas-dev/pandas/blob/master/pandas/core/array_algos/putmask.py#L58-L77

Reply:
@hirosassa Thank you for the comment. I've added a docstring.

Reply:
Looks good! Thank you!